contrib/release.sh

```diff
@@ -118,66 +120,57 @@ popd
 
 # Build Windows.
 pushd $basedir/dgraph/dgraph
-  env GOOS=windows GOARCH=amd64 go get -v -d .
-  env GOOS=windows GOARCH=amd64 GO111MODULE=on go build -v -o dgraph-windows-amd64.exe -ldflags \
+  xgo -image="$XGO_IMAGE" --targets=windows/amd64 -ldflags \
```

Comment:
Is there any benefits to doing this in this way instead of actually compiling on Windows with a CI?